src/applications/ivc-champva/10-7959C/chapters/healthInsuranceInformation.js

```diff
@@ -13,7 +13,6 @@ import {
   checkboxGroupSchema,
 } from 'platform/forms-system/src/js/web-component-patterns';
 import { requiredFiles } from '../config/constants';
-import { applicantWording } from '../../shared/utilities';
```

Comment:
This was a straggler left over from removing the list loop. We don't use this function anymore.